pdns/ws-auth.cc

```diff
@@ -1248,6 +1250,7 @@ static void apiZoneCryptokeysPUT(DNSName zonename, int inquireKeyId, HttpRequest
   auto document = req->json();
   //throws an exception if the key does not exist or is not a bool
   bool active = boolFromJson(document, "active");
+  bool published = boolFromJson(document, "published");
```

Comment:
This looks like an incompatible API change.

Reply:
Made it true by default so its backwards compatible.